src/Uno.UI/Microsoft/UI/Xaml/Controls/TreeView/TreeViewList.cs

```diff
@@ -125,12 +125,22 @@ private void OnContainerContentChanging(object sender, ContainerContentChangingE
 					treeViewItem.SetItemsSource(targetNode, itemsSource);
 				}
 			}
+#if HAS_UNO
```

Comment:
Please include a comment which issue is this working around if possible for clarity